File: Python/SQLAlchemy_ORM/dal/entities/shipper.py
from sqlalchemy import Column, Integer, String
from sqlalchemy.orm import relationship
from dal.data.base import Base

class Shipper(Base):
    __tablename__ = 'Shippers'

    ShipperID = Column(Integer, primary_key=True)
    CompanyName = Column(String(40))
    Phone = Column(String(24))

    # Relationship with Orders
    orders = relationship('Order', back_populates='shipper')

File: Go/GORM_ORM/dal/entities/shipper.go
package entities

type Shipper struct {
	ShipperID   int     `gorm:"primaryKey;autoIncrement;column:ShipperID"`
	CompanyName string  `gorm:"size:40;column:CompanyName"`
	Phone       *string `gorm:"size:24;column:Phone"`
	Orders      []Order `gorm:"foreignKey:ShipVia"`
}
